.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: 🐛 Bug report
+description: Create a report to help us reproduce and fix a bug
+labels: [bug]
+
+body:
+- type: textarea
+  attributes:
+    label: Description
+    description: >
+      Please provide a clear and concise description of the bug.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps to reproduce
+    description: >
+      We can't solve your issue if we can't reproduce it. Please provide a
+      [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)
+      that shows how to reproduce the bug. If the bug requires any additional files to
+      reproduce, please upload those or provide a download link. Your code should be
+      runnable and include all relevant imports.
+    placeholder: |
+      1. Download any Landsat 8 scene from https://earthexplorer.usgs.gov/
+      2. Run the following code to reproduce the error
+
+      ```python
+      from torchgeo.datasets import Landsat8
+
+      ds = Landsat8("/path/to/downloads/directory")
+      ...
+      ```
+  validations:
+    required: true
```

Comment:
My only concern is making the reproducible example required. I don't want someone to _not_ report something just because they _have_ to provide a reproducible example. Maybe we can explain that bugs are much easier to fix if there is reproducible code and we don't have a lot of free time to make our own examples?

Reply:
I think having a _reproducible_ example is a low bar. At the bare minimum, just upload the code you ran and the files you ran it on to produce the error message. Having a _minimal_ reproducible example is a bigger hurdle, but users rarely consider that to be a requirement unfortunately. If someone reported an issue with just an error message and didn't tell me how they got that error message, I would probably just close the issue. I would go so far as to say having a reproducible example is the _only_ thing that is absolutely required to report an issue.

Reply:
That's totally fine -- just a minor concern! The more I think about it the more I agree, we really don't have a ton of time to dig into things (esp. if they are obscure).